src/molecules/Table/Table.test.tsx

```diff
@@ -0,0 +1,124 @@
+import { mount, ReactWrapper, ShallowWrapper } from 'enzyme';
+import React from 'react';
+import renderer from 'react-test-renderer';
+import 'jest-styled-components';
+
+import { Table } from './Table';
+import { Badge } from 'atoms';
+
+describe('Table', () => {
+  let wrapper: ReactWrapper | ShallowWrapper;
+
+  const args = {
+    columns: [
+      {
+        Header: 'Column 1',
+        accessor: 'col1' // accessor is the "key" in the data
+      },
+      {
+        Header: 'Column 2',
+        accessor: 'col2'
+      },
+      {
+        Header: 'Column 3',
+        accessor: 'col3'
+      }
+    ],
+    data: [
+      {
+        col1: <img src="https://picsum.photos/75" />,
+        col2: 'Row 1 Cell 2',
+        col3: 'Row 1 Cell 3'
+      },
+      {
+        col1: 'Row 2 Cell 1',
+        col2: <a href="#">Link</a>,
+        col3: 'Row 2 Cell 3'
+      },
+      {
+        col1: 'Row 3 Cell 1',
+        col2: 'Row 3 Cell 2',
+        col3: <Badge status="neutral">Badge</Badge>
+      }
+    ]
+  };
+
+  beforeEach(() => {
+    wrapper = mount(<Table columns={args.columns} data={args.data} />);
+  });
+
+  describe('basic table structure', () => {
+    it('renders the table', () => {
+      expect(wrapper.find('table[role="table"]').exists()).toBeTruthy();
+      expect(wrapper.find('thead').exists()).toBeTruthy();
+      expect(wrapper.find('tbody[role="rowgroup"]').exists()).toBeTruthy();
+      expect(
+        wrapper.find('table thead tr th PureIcon[icon="unfold"]')
+      ).toHaveLength(args.columns.length);
+    });
+  });
+
+  describe('unsorted table data', () => {
+    // column headers
+    it('renders the header content', () => {
+      const len = args.columns.length;
+      const columns = Array.from(Array(len), (x, index) => index);
+
+      columns.forEach(column => {
+        const text = args.columns[column].Header;
+        expect(wrapper.find('table thead tr th').at(column)).toHaveText(text);
+      });
+    });
+
+    it('renders the unsorted table content', () => {
+      const component = renderer
+        .create(<Table columns={args.columns} data={args.data} />)
+        .toJSON();
+      expect(component).toMatchSnapshot();
```

Comment:
This should work with `mount` from enzyme now, so you can get rid of `renderer` import.
```suggestion
      const wrapper = mount(<Table columns={args.columns} data={args.data} />);
      expect(wrapper).toMatchSnapshot();
```

Reply:
This change is breaking the test in the `sorted table data` block.

Reply:
I reverted this in all places since it added hundreds of thousands of lines of code to the snapshot file.